CHANGES.rst

```diff
@@ -172,6 +172,9 @@ astropy.io.ascii
 astropy.io.fits
 ^^^^^^^^^^^^^^^
 
+- Properly handle opening of FITS files from HTTPResponses from remote URLs.
+  [#6398]
```

Comment:
Please fix the typo here in the PR number. Also while you're at it, move the entry down to the 2.0.1 section.